packages/Modal/src/index.js

```diff
@@ -0,0 +1,12 @@
+import Modal from "./Modal";
+import FocusTrap from "./components/FocusTrap";
+import Header from "./components/Header";
+import Content from "./components/Content";
+import Footer from "./components/Footer";
+
+Modal.FocusTrap = FocusTrap;
```

Comment:
Same I think we should use `<Modal.Overlay />` and keep the same pattern as the sidepanel.

Reply:
> keep the same pattern as the sidepanel

What do you mean by this?